### PR TITLE
Handle race condition in shared classes defineClass()

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Handle IllegalArgumentException when parallel classloaders define the same
package. cf. eclipse/openj9#3038

Signed-off-by: Peter Bain <peter_bain@ca.ibm.com>

### CODE REVIEW

File: src/java.base/share/classes/java/net/URLClassLoader.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 1997, 2017 All Rights Reserved
+ * (c) Copyright IBM Corp. 2000, 2018 All Rights Reserved
```
Comment (pshipton) [inline src/java.base/share/classes/java/net/URLClassLoader.java]: Why the change in the start date?

**Comment by pshipton** [comment]
jenkins test sanity zlinux

File: src/java.base/share/classes/java/net/URLClassLoader.java (hunk 1 above)
Comment (pdbain-ibm) [reply src/java.base/share/classes/java/net/URLClassLoader.java]: Copy/paste error.

**Comment by pshipton** [comment]
PR testing is here https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-Sanity-JDK11-linux_390-64_cmprssptrs-OpenJDK11/10/
